File: timus/python/p1785.py
# Problem 1785. Lost in Localization
# http://acm.timus.ru/problem.aspx?space=1&num=1785

n = int(raw_input())

if n <= 4:
	print "few"
elif n >= 5 and n < 10:
	print "several"
elif n >= 10 and n < 20:
	print "pack"
elif n >= 20 and n < 50:
	print "lots"
elif n >= 50 and n < 100:
	print "horde"
elif n >= 100 and n < 250:
	print "throng"
elif n >= 250 and n < 500:
	print "swarm"
elif n >= 500 and n < 1000:
	print "zounds"
else:
	print "legion"

File: timus/java/p1785.java

import java.util.Scanner;

/**
 * Lost in Localization
 * @author Milena Araujo
 *
 */

public class p1785 {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int n = input.nextInt();
		
		if(n >= 1 && n <= 4){
			System.out.println("few");
		}
		
		if(n >= 5 && n <= 9){
			System.out.println("several");
		}
		
		if(n >= 10 && n <= 19){
			System.out.println("pack");
		}
		
		if(n >= 20 && n <= 49){
			System.out.println("lots");
		}
		
		if(n >= 50 && n <= 99){
			System.out.println("horde");
		}
		
		if(n >= 100 && n <= 249){
			System.out.println("throng");
		}
		
		if(n >= 250 && n <= 499){
			System.out.println("swarm");
		}
		
		if(n >= 500 && n <= 999){
			System.out.println("zounds");
		}
		
		if(n >= 1000){
			System.out.println("legion");
		}
		
	}
	
}
